### PR TITLE
docs: 🛠 Fixed typo found during translation

### PR DESCRIPTION
**Pull request by dance2die**
I found a typo while translating CLI Usage

Doc: https://allcontributors.org/docs/en/cli/usage#all-contributors-init
Crowdin - https://crowdin.com/translate/all-contributors/30/en-ko

Thought I'd make a PR before I forget

### CODE REVIEW

File: docs/cli/usage.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Use `check` to compare contributors from GitHub with the ones credited in your
 it's due.
 
 > Due to GitHub API restrictions, this command only works for projects with less
-> than 500 contributors. (Unless you set a `PRIVATE_TOKEN`) as mentioned [below in GitHub users](#github-users)
+> than 500 contributors (Unless you set a `PRIVATE_TOKEN` as mentioned [below in GitHub users](#github-users)).
```
Comment (dance2die) [inline docs/cli/usage.md]: Found here -> https://allcontributors.org/docs/en/cli/usage#all-contributors-check

**Comment by jakebolam** [comment]
Thanks @dance2die 🎉

**Comment by dance2die** [comment]
Thank you for the review 👨‍🎓 & merge~ 🥅

**Comment by dance2die** [comment]
@allcontributors please add @dance2die for doc

**Pull request merged**